### PR TITLE
Add resources

### LINKED ISSUE #1033: [Migration] Adapt migration process modifications with the integration test
Extend the scope of the ui integration test to cover migration process modifications related to issues #906 and #891 (`SESSION_USER` replacements  and modifications to analytic privilege `modelUri` and `whereSql` elements) by adding the needed files that need to be modified to the tested project and to the expected results project in the test resources.

Example files which could be used [migration-files.zip](https://github.com/SAP/xsk/files/7898476/migration-files.zip)

Related to issue #992

### PR DESCRIPTION
**Pull request by ivanvolkoff**
Resources for integration test. Fixes #1033

### CODE REVIEW

**Review by vmutafov — APPROVED** [review]

File: integration-tests/ui/src/test/resources/migration/MIGR_TOOLS/xsk-test-app/xsk-test-app-copy.hdi [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
```
Comment (ThuF) [inline integration-tests/ui/src/test/resources/migration/MIGR_TOOLS/xsk-test-app/xsk-test-app-copy.hdi:1]: Why the filename is `xsk-test-app-copy.hdi`, shouldn't it be `xsk-test-app.hdi`?

File: integration-tests/ui/src/test/resources/migration/MIGR_TOOLS/xsk-test-app/xsk-test-app-copy.hdiconfig [hunk 2]
```diff
@@ -0,0 +1,16 @@
+{
```
Comment (ThuF) [inline integration-tests/ui/src/test/resources/migration/MIGR_TOOLS/xsk-test-app/xsk-test-app-copy.hdiconfig:1]: Why the filename is `xsk-test-app-copy.hdiconfig`? it seems that in the `*.hdi` file it's referenced as `xsk-test-app.hdi`.

**Review by ThuF — CHANGES_REQUESTED** [review]
Why `xsk-test-app/images/test/test.html` and `xsk-test-app/images/test2/test2.xsjs` are empty? Additional comments about the name of the `*.hdi` and `*.hdiconfig` files.

File: integration-tests/ui/src/test/resources/migration/MIGR_TOOLS/xsk-test-app/xsk-test-app-copy.hdiconfig (hunk 2 above)
Comment (ivanvolkoff) [reply integration-tests/ui/src/test/resources/migration/MIGR_TOOLS/xsk-test-app/xsk-test-app-copy.hdiconfig:1]: copy/paste omission

**Comment by ivanvolkoff** [comment]
> Why `xsk-test-app/images/test/test.html` and `xsk-test-app/images/test2/test2.xsjs` are empty? Additional comments about the name of the `*.hdi` and `*.hdiconfig` files.

Both files are added just to test if migration is 'migrates' files from nested folders. I will add some content to these files.

**Review by ThuF — APPROVED** [review]
+1

**Pull request merged**